spp_scan_id_document/tests/test_registrant.py

```diff
@@ -38,4 +38,3 @@ def test_on_scan_id_document_details_success(self):
 
         self.assertEqual(self.applicant.family_name, "Red")
         self.assertEqual(self.applicant.given_name, "Blue")
-        self.assertEqual(self.applicant.gender, "Male")
```

Comment:
@reichie020212 why did you remove this line? I meant: the data is there, it should work 😕

Reply:
@nhatnm0612 to be honest, I don't know why this test is not working on the job here in the link (https://github.com/OpenSPP/openspp-registry/actions/runs/4980432358/jobs/8913336844?pr=235) but when I run it in my local it is working. That's the reason why I decided to just remove this line of code.

Reply:
@reichie020212 `father_name` is defined in another module [which depends on this module]. On this test run, test cases start right after the module installed and at that time, `res_partner` table does not have this column yet, also Individual instance does not recognize this field yet to write. I think you should remove this logic out of PR, coz `pds-spp` is not our product, this one is.

Reply:
@nhatnm0612 noted on that

Reply:
@nhatnm0612 done updating the code. kindly check again. Thanks.